### PR TITLE
B2CQA 1302 - Add Allure Reporting to Playwright and Desktop test workflow

### PR DESCRIPTION
**Pull request by ggilchrist-ledger**
### 📝 Description

- Add `allure-playwright` reporter for PW tests
- Add report upload steps for `test-desktop.yml`

### ❓ Context

- **Impacted projects**: `` 
- **Linked resource(s)**: `` 

### ✅ Checklist

- [ ] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: apps/ledger-live-desktop/package.json [hunk 1]
```diff
@@ -184,6 +184,7 @@
     "@typescript-eslint/eslint-plugin": "^5.28.0",
     "@typescript-eslint/parser": "5.28.0",
     "@vitejs/plugin-react": "^2.0.0",
+    "allure-playwright": "2.0.0-beta.19",
```
Comment (valpinkman) [inline apps/ledger-live-desktop/package.json:187]: why is the version fixed ? is 2.0.0 out of beta yet ?

**Review by valpinkman — CHANGES_REQUESTED** [review]

File: .github/workflows/test-desktop.yml [hunk 2]
```diff
@@ -256,3 +256,58 @@ jobs:
...
+        uses: actions/download-artifact@v2
+        with:
+          name: allure-results-macos
+          path: allure-results-macos
+      - name: Download Allure Report - Linux
+        uses: actions/download-artifact@v2
+        with:
+          name: allure-results-linux
+          path: allure-results-linux
+      - name: Send Results and Generate Allure Report - Windows
```
Comment (abdurrahman-ledger) [inline .github/workflows/test-desktop.yml:281]: You can use this composite [action](https://github.com/LedgerHQ/ledger-live/blob/develop/tools/actions/composites/upload-allure-report/action.yml) (to improve) for sending action to factorise a 'little' bit.

Comment (ggilchrist-ledger) [reply .github/workflows/test-desktop.yml:313]: Cool will take a look

**Review by valpinkman — APPROVED** [review]

File: .pnpmfile.cjs [hunk 3]
```diff
@@ -154,7 +154,11 @@ function readPackage(pkg, context) {
         "expo-asset": "*",
       }),
       /* Other packages */
-      addDependencies("detox", { "@jest/reporters": "*", "jest-environment-node": "*", "jest-circus": "*" }),
+      addDependencies("detox", {
```
Comment (ggilchrist-ledger) [inline .pnpmfile.cjs:157]: Just a prettier change

**Pull request merged**